actions/actionsMatchtxt_test.go

```diff
@@ -84,35 +88,35 @@ Your system has not yet been tuned. Please visit `+"`"+`saptune note`+"`"+` and
 
 var PrintHelpAndExitMatchText = `saptune: Comprehensive system optimisation management for SAP solutions.
```

Comment:
is something that make sense to extract this multiline strings into a 1 single function inside actions.go? it can prevent to do a copy paste and human errors because you change only 1 time in 1 place!

Reply:
yes, wonderful idea. Hope the implementation is good enough :smile:  See new committed code